airtime_mvc/application/views/scripts/index/index.phtml

```diff
@@ -306,21 +270,12 @@
 <script type="text/javascript">
     $(document).ready(function () {
 
-
-
         $("#player_iframe").load(function () {
 
             $("#player_iframe").contents().find('.bottom_bar').append("<div class='flex_spacer' data-tab='tab-1'></div>");
             $("#player_iframe").contents().find('.bottom_bar').append("<div class='schedule_btn button current' data-tab='tab-1'></div>");
             $("#player_iframe").contents().find('.bottom_bar').append("<div class='about_us_btn button' data-tab='tab-2'></div>");
-
-            <?php
-            // Is there a better way to do this?
-            if ($this->displayRssTab) {?>
-                $("#player_iframe").contents().find('.bottom_bar').append("<div class='station_rss_btn button' data-tab='tab-4'></div>");
-            <?php
-            }
-            ?>
+            $("#player_iframe").contents().find('.bottom_bar').append("<div class='station_rss_btn button' data-tab='tab-4'></div>");
```

Comment:
`displayRssTab` is still getting set in `IndexController`:
https://github.com/LibreTime/libretime/blob/6c4b1b5e71f8de260a1b04ee3f04c7489b1832bd/airtime_mvc/application/controllers/IndexController.php#L83
It is managed in `PreferenceController`:
https://github.com/LibreTime/libretime/blob/6c4b1b5e71f8de260a1b04ee3f04c7489b1832bd/airtime_mvc/application/controllers/PreferenceController.php#L111
It looks like this change will remove the ability to hide the rss "tab" or am I missing something any you found "a better way to do this"?

Reply:
Thanks for mentioning it was in the controllers. There doesn't appear to be a setting for it anywhere in the interface, so I removed the logic for it.

Reply:
it's this thing on `/podcast/station`:

![image](https://user-images.githubusercontent.com/116588/103171847-99824600-484f-11eb-8d50-80cb53a67003.png)

Reply:
Gotcha